doc/api/addons.markdown

```diff
@@ -868,6 +939,19 @@ Test in JavaScript by running:
     // test.js
     const addon = require('./build/Release/addon');
 
+## Native Abstractions for Node.js
```

Comment:
I wouldn't mind this being moved up nearer to the top so that people understand they can go to node-addon-examples to find NANified versions of the examples rather than jumping straight into direct-V8 code. The NAN documentation is also really extensive and contains a ton of useful information beyond what you can glean from here or the V8 docs so it'd be worth pointing that out.